core/kazoo_voicemail/src/kvm_message.erl

```diff
@@ -239,7 +239,9 @@ set_folder(Folder, Message, AccountId) ->
     end.
 
 -spec maybe_set_folder(kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary(), kz_json:object()) -> db_ret().
```

Comment:
Can you please fix the spec here, the second argument should be `vm_folder()`.

Reply:
Yes, we will update for this.